content/en/Changelog/_index.md

```diff
@@ -0,0 +1,21 @@
+
+---
+title: "Changelog"
+linkTitle: "Changelog"
+weight: 30
+menu:
+  main:
+    weight: 30
+---
+
+{{% pageinfo %}}
+This page includes notable external changes to Cobalt software.
+
+The format is based on guidance from [Keep a Changelog](http://keepachangelog.com/).
+
+This changelog includes two types of versioning:
+
+- Product: [Calendar Versioning](https://calver.org)
+- API: [Semantic Versioning](http://semver.org/)
+{{% /pageinfo %}}
+
```

Comment:
Perhaps you'll want to bootstrap this file with a first changelog entry so that it's easier for developers to edit the file on top of that (and they know where to place the new entries).

Reply:
Good idea, but it's sort of a "chicken and egg" question... Assuming I get the OK, I'm thinking, push this as a template, and then do the first couple of (weekly) entries myself.